service/src/main/java/cz/muni/fi/pa165/service/UserServiceImpl.java

```diff
@@ -0,0 +1,150 @@
+package cz.muni.fi.pa165.service;
+
+import cz.fi.muni.pa165.dao.UserDao;
+import cz.fi.muni.pa165.entity.User;
+import cz.fi.muni.pa165.enums.Gendre;
+import cz.fi.muni.pa165.enums.Role;
+import org.springframework.stereotype.Service;
+
+import javax.crypto.SecretKeyFactory;
+import javax.crypto.spec.PBEKeySpec;
+import javax.inject.Inject;
+import java.math.BigInteger;
+import java.security.NoSuchAlgorithmException;
+import java.security.SecureRandom;
+import java.security.spec.InvalidKeySpecException;
+import java.util.List;
+
+@Service
+public class UserServiceImpl implements UserService {
+
+    @Inject
+    private UserDao userDao;
+
+    @Override
+    public User findByEmail(String email) {
+        return null;
```

Comment:
chybí implementace, null by to vracet nemelo

Reply:
taky jsem to napsal do komentare, protoze jsem myslel ze to prijde jeste do dao. mam teda metodu pridat do dao + test nebo ji primo implementovat sem?

Reply:
pridat do dao + test